### PR TITLE
Initialize /portfolio and /balance routes

### PR DESCRIPTION
**Pull request by harshitsinghai77**

### CODE REVIEW

File: app/api/router/tokens.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
+from typing import List
 from schemas.TokenBalance import TokenBalance
 
 router = APIRouter()
 
 
 @router.get("/tokens/{chain}/{address}")
-async def tokens(chain: int, address: str) -> [TokenBalance]:
+async def tokens(chain: int, address: str) -> List[TokenBalance]:
```
Comment (shreyaspapi) [inline app/api/router/tokens.py:11]: Ohh we need this for `Docs`? - `List`?

File: app/api/router/trader.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+from web3 import Web3
+from typing import List
+
+from schemas.Trader import TraderAddress, Portfolio
+from api.utils.trader import formatUnits, formatEther, SPOT_PRICE, getLiquidationPrice
+
+trader_route = APIRouter()
+
+BASE_PATH = "api/artifacts/portfolio/"
+STAGE_NAME = "production"
```
Comment (shreyaspapi) [inline app/api/router/trader.py:15]: Env var?

File: app/main.py [hunk 3]
```diff
@@ -15,5 +16,6 @@
 
 # Declare Routers
 app.include_router(tokens.router)
+app.include_router(trader_route, prefix="/trader")
```
Comment (shreyaspapi) [inline app/main.py:19]: Don't add prefix here, check tokens.router declaration that's better practice.

**Review by shreyaspapi — APPROVED** [review]

File: app/api/router/tokens.py (hunk 1 above)
Comment (harshitsinghai77) [reply app/api/router/tokens.py:11]: pylint suggestion. It improves code readability.

File: app/api/router/trader.py (hunk 2 above)
Comment (harshitsinghai77) [reply app/api/router/trader.py:16]: `STAGE_NAME`? Yeah, we can do this.

**Pull request merged**